vta/tutorials/autotvm/tune_relay_vta.py

```diff
@@ -357,7 +357,7 @@ def tune_and_evaluate(tuning_opt):
     )
 
     # filter out non-packed conv2d task
-    tasks = list(filter(lambda t: len(t.args[0][1]) > 4, tasks))
+    tasks = list(filter(lambda t: len(t.args[0][1]) > 4 and "conv" in t.name, tasks))
```

Comment:
what happened here, did this transpose change introduce a new task or something?

Reply:
yes

Reply:
actually, no, but this check makes sure anyways.

Reply:
Isn't the new added schedule not tunable? Or is there any concern of adding knobs?

Reply:
We may want to tune it in the future.